packages/angstrom/angstrom.0.1.0/opam

```diff
@@ -10,26 +10,22 @@ build: [
     "ocaml"
     "setup.ml"
     "-configure"
-    "--prefix"
-    prefix
+    "--prefix" prefix
     "--%{base-unix:enable}%-unix"
     "--%{lwt:enable}%-lwt"
     "--%{async:enable}%-async"
+    "--enable-tests" {with-test}
   ]
   ["ocaml" "setup.ml" "-build"]
-  ["ocaml" "setup.ml" "-configure" "--enable-tests"] {with-test}
```

Comment:
what's this removal fixing? If I remember oasis right, the tests had to be explicitly enabled to work

Reply:
yes this is just move to the line above. The opam file has redundant instructions for configure and build which came from a wrong assumptions about how the `with-test` variable works. This is just a simplification/ci-optimisation